.github/workflows/ci.yml

```diff
@@ -103,8 +103,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
```

Comment:
Did you run into dependency issues with 3.6?

Reply:
Python3.6 support is going to be removed soon: https://nautobot.readthedocs.io/en/stable/release-notes/version-1.2/#v124-2022-01-13 but it is still supported https://github.com/nautobot/nautobot/blob/develop/.github/workflows/ci.yml#L86

Reply:
3.6 was failing poetry update and I copied the pyproject.toml from the SSOT plugin which removed 3.6  https://github.com/nautobot/nautobot-plugin-ssot-ipfabric/blob/develop/pyproject.toml

Because `nautobot = "^1.2.0"` was not mentioned in the pyproject.toml every time I built the project it would roll back nautobot to version 1.1.5.  Once I added this then python 3.6 didn't like it.

Build: ` Updating nautobot (1.2.6b1 -> 1.1.5)`

Poetry update errors:
` Because no versions of celery match >5.1.0,<5.1.1 || >5.1.1,<5.1.2
   and celery (5.1.0) depends on click (>=7.0,<8.0), celery (>=5.1.0,<5.1.1 || >5.1.1,<5.1.2) requires click (>=7.0,<8.0).
  And because celery (5.1.1) depends on click (>=7.0,<8.0), celery (>=5.1.0,<5.1.2) requires click (>=7.0,<8.0).
  Because nautobot (1.2.1) depends on celery (>=5.1.0,<5.2.0)
   and nautobot (1.2.2) depends on celery (>=5.1.0,<5.2.0), nautobot (1.2.1 || 1.2.2) requires celery (>=5.1.0,<5.2.0).
  Thus, nautobot (1.2.1 || 1.2.2) requires click (>=7.0,<8.0) or celery (>=5.1.2,<5.2.0).
  And because no versions of nautobot match >1.2.1,<1.2.2 || >1.2.2,<1.2.3 || >1.2.3,<1.2.4 || >1.2.4,<1.2.5 || >1.2.5,<2.0.0
   and nautobot (1.2.4) depends on celery (>=5.1.2,<5.2.0), nautobot (>=1.2.1,<1.2.3 || >1.2.3,<1.2.5 || >1.2.5,<2.0.0) requires click (>=7.0,<8.0) or celery (>=5.1.2,<5.2.0).
  And because nautobot (1.2.3) depends on celery (>=5.1.2,<5.2.0)
   and nautobot (1.2.5) depends on celery (>=5.1.2,<5.2.0), nautobot (>=1.2.1,<2.0.0) requires click (>=7.0,<8.0) or celery (>=5.1.2,<5.2.0).
  Because celery (5.1.2) depends on click (>=7.0,<8.0)
   and no versions of celery match >5.1.2,<5.2.0, celery (>=5.1.2,<5.2.0) requires click (>=7.0,<8.0).
  Thus, nautobot (>=1.2.1,<2.0.0) requires click (>=7.0,<8.0).
  Because no versions of black match !=22.1.0
   and black (22.1.0) depends on click (>=8.0.0), every version of black requires click (>=8.0.0).
  Thus, black is incompatible with nautobot (>=1.2.1,<2.0.0).
  So, because nautobot-chatops-ipfabric depends on both nautobot (^1.2.1) and black (*), version solving failed.`

Reply:
I'm okay with dropping 3.6.